docs/config.yml

```diff
@@ -31,3 +31,9 @@ security:
     - '^HUGO_'
     - 'VERSIONS'
     - 'CURRENT_VERSION'
+
+go_initialisms:
+  replace_defaults: true
+  initialisms:
+    - 'ID'
```

Comment:
Flipping to false above means this is not necessar.

Reply:
Done!